README.md

```diff
@@ -17,47 +19,114 @@ Currently, we don't typically assign contributors. If you see an open issue that
 
-3. Install dependencies
+### 3. Install dependencies
 
 ```
 make install
 ```
 
-4. Start a server on localhost to see your changes
+### 4. Start a server on localhost to see your changes
+
+In api.py, comment out
+```
+CORS(app)
+```
+
+Add
+```
+CORS(app, resources={r"/*": {"origins": "*"}})
+```
+
+Then run:
 
 ```
 make debug
 ```
 
 Now you're ready to start developing!
 
-## Testing, Formatting, Changelogging
+NOTE: If you are using Airpods or other Apple bluetooth products, you may get an error related to the port. If this is the case, define a specific port in the debug statement in the Makefile. For example:
+```
+debug:
+	FLASK_APP=policyengine_api.api FLASK_DEBUG=1 flask run --without-threads --port=5001
+```
 
-You've finished your contribution, but now what? Before opening a PR, we ask contributors to do three things.
+A simple API get call you can send in Postman to make sure it is working as expected is:
+```
+http://127.0.0.1:5001/us/policy/2
+```
 
-### Step 1: Testing
-To test your changes against our series of automated tests, run
+### 5. To test in combination with policyengine-app: 
 
+1. In policyengine-app/src/api/call.js, comment out
 ```
-make debug-test
+const POLICYENGINE_API = "https://api.policyengine.org";
 ```
+And add
+```
+const POLICYENGINE_API = "http://127.0.0.1:5001" (or the relevant port where the server is running)
+```
+2. Start server as described above
+3. Start app as described in policyengine-app/README.md
+
+NOTE: Any output that needs to be calculated will not work. Therefore, only household output can be tested with this setup.
+
+### 6. Testing calculations
+
+To test anything that utilizes Redis or the API's service workers (e.g. anything that requires calculations), you'll also need to complete the following steps:
 
-If testing anything that utilizes Redis (unlikely) or the API's service workers (very unlikely), you'll also need to run one of the following:
+1. Start Redis
+- Install Redis:
+```
+brew install redis
+```
+- Start Redis:
 ```
 redis-server
 ```
-or 
+2. Start the API service worker
+- This requires getting the POLICYENGINE_DB_PASSWORD from a teammate
+- A teammate also must add you as a Google Cloud user and you will need to configure [Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc#how-to) using your Google Account. Then complete the following steps:
+
+3. Install Google Cloud CLI as described in the ADC documentation
+
+4. Run the following to initialize it
+```
+gcloud init
+```
+When prompted "Pick cloud project to use:" Select the number that corresponds to "policyengine-api"
+When prompted "Do you want to configure a default Compute Region and Zone? (Y/n)?" Select 'n'
+
+Create the credentials: 
+```
+gcloud auth application-default login
+```
+
+Return to policyengine-api folder and relevant python environment and run:
+```
+POLICYENGINE_DB_PASSWORD="" python policyengine_api/worker.py
```

Comment:
This will sound super nitpicky, but we've had issues in the past, so you might want to write something like "PASSWORD_HERE" inside the quotes